pkgs/tools/misc/lfimg/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenvNoCC
+, lib
+, fetchFromGitHub
+, ueberzug
+, lf
+, makeWrapper
+, ffmpegthumbnailer
+, imagemagick
+, wkhtmltopdf
+, epub-thumbnailer
+, poppler
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "lfimg";
+  version = "87b1f6b";
+
+  src = fetchFromGitHub {
+    owner = "cirala";
+    repo = pname;
+    rev = version;
+    sha256 = "0ykjhvl3yd8mwaf1vpz6wa0zph7ahn5pw88dk8vy62wm0d6aajgg";
+  };
+
+  dontBuild = true;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm755 lfrun $out/bin/lfrun
```

Comment:
This might also help fix your issues with it not working properly

Reply:
Thx for the review again @06kellyjac ! The fact is that it is working even without the patched preview and cleaner, but only with the chafa backend, I could not use ueberzug. I tested both on wayland and x11. I dont think it is necessary to have a patched preview and cleaner since they both should probably have access to the same PATH as the patched lfrun.